Safor/index.html

```diff
@@ -0,0 +1,160 @@
+                            factors and signals for help.</p>
+                    </li>
+                    <li>
+                        <img src="img/five/Rectangle_6_Copy.png">
+                        <p>Business proposals, external and internal requests and new technologies.
+                            Transparency of all processes of interaction with our employees, customers 
+                            and business partners.</p>
+                    </li>
+                    <li>
+                        <img src="img/five/Polygon_Copy.png">
+                        <p>In making decisions, the quality of the services provided and the distribution 
+                            of funds. We use all available resources to achieve maximum results in 
+                            our work and communication.</p>
+                    </li>
+                </ul>
+            </div>
+            <div class="five_two">
+                <h2 class="five_two_headline">FOR PEOPLE</h2>
+                <ul>
+                    <li>
+                        <img src="img/five/Polygon.png">
+                        <p>Do not waste time to explain your location and nature of the threat, guard 
+                            patrols get your alarm in 1 second and advances to you no matter whether you 
+                            are in motion or not.</p>
+                    </li>
+                    <li>
+                        <img src="img/five/Star.png">
+                        <p>You will be foundregardless of whether you are in town or on a trip.
+                            The service covers the entire territory of the country and allows you to specify
+                            the exact location even when in a crowded place</p>
+                    </li>
+                    <li>
+                        <img src="img/five/Rectangle_6_Copy_2.png">
+                        <p>And we will help in any situation. Only after coming to you the calling
+                            will be closed by the patrol arrived, whose goal is to protect and help you.</p>                    
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="six">
+            <h1 class="six_headline">HAVE QUESTIONS? CONTACT US</h1>
+            <form class="six_inputs" action="#">
+                <input type="text" placeholder="NAME">
+                <input type="text" placeholder="EMAIL">
+                <input type="text" placeholder="MESSAGE">
+                <input type="submit" value="SEND">
+            </form>
+        </section>
+        <section class="footer">
+            <img src="img/footer/Group.svg">
+            <p>© SAFOR 2018 All Rights Resevered 85 Great Portland Street, First Floor London, W1W7LT, United Kingdom</p>
+            <div>
+                <a href="#"><img src="img/footer/1.svg"></a>
+                <a href="#"><img src="img/footer/2.svg"></a>
```

Comment:
On resolution 375, size of instagram icon incorrect. Also check distance between icons.

Reply:
Try to use another instagram and maybe another social icons on lower resolution.